automatic/git.install/tools/helpers.ps1

```diff
@@ -0,0 +1,69 @@
+function Get-InstallKey() {
+    $registryKeyName = 'Git_is1'
+    $installKey = "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$registryKeyName"
+    if ((Get-ProcessorBits 64) -and $env:chocolateyForceX86 -ne 'true') {
```

Comment:
Yeah its wrong, but not in above but order `$installKey` values should be ordered the opposite way.